command/genesis.go

```diff
@@ -252,6 +264,40 @@ func (c *GenesisCommand) Run(args []string) int {
 		}
 	}
 
+	if len(prestake) != 0 {
```

Comment:
We need to check if prestake is not empty here? because you just run for-loop inside if block and for-loop will never run any loop if prestake is nil or empty slice.

Reply:
You're actually right. I'll remove it, along with the same code block for `premine` 👍